src/Umbraco.Web.UI.Client/src/views/components/application/umb-login.html

```diff
@@ -2,10 +2,10 @@
 
     <div id="login" class="umb-modalcolumn umb-dialog" ng-class="{'show-validation': vm.loginForm.$invalid}" ng-cloak>
 
-        <div class="login-overlay__background-image" ng-if="vm.backgroundImage" ng-style="{'background-image':'url(' + vm.backgroundImage + ')'}"></div>
+        <div class="login-overlay__background-image" ng-style="{'background-image': 'url('+vm.backgroundImage+')'}"></div>
```

Comment:
If no background image is set (in the Umbraco settings), we should not render the overlay `div` (this also prevents an empty/invalid background-image to be set).
```suggestion
        <div class="login-overlay__background-image" ng-if="vm.backgroundImage" ng-style="{'background-image': 'url('+vm.backgroundImage+')'}"></div>
```